docs/docs/resources/architecture/components-hierarchy.md

```diff
@@ -1,20 +1,20 @@
 ```mermaid
 flowchart BT
     KubernetesApp --> PipelineComponent
-    KafkaConnector --> PipelineComponent
     KafkaApp --> KubernetesApp
     StreamsApp --> KafkaApp
     ProducerApp --> KafkaApp
+    KafkaConnector --> PipelineComponent
     KafkaSourceConnector --> KafkaConnector
     KafkaSinkConnector --> KafkaConnector
-    
-    click KubernetesApp "../kubernetes-app"
```

Comment:
does it have to be relative link? Ideally, I'd wanna do something like /user/core-concepts/... but it doesn't work and setting it to /kpops/user/core-concepts/... breaks the docs version history

Reply:
How does it break the version history?

Reply:
Why don't you want it as relative links?

Reply:
They result in 404 currently

Reply:
When did they break?

Reply:
I suppose during restructuring https://github.com/bakdata/kpops/pull/329

Reply:
I can't reproduce the 404 bug. When I go to this link and click the components it takes me to the correct page 🤔 
https://bakdata.github.io/kpops/2.0/user/core-concepts/components/overview/

Reply:
it's from this page https://bakdata.github.io/kpops/2.0/resources/architecture/components-hierarchy

Reply:
perhaps we should simply exclude resources from the search?

Reply:
Yeah, I am pretty sure that this is not supposed to be seen on its own.